### PR TITLE
User profile first pass

### PR DESCRIPTION
**Pull request by ajacksified**
First pass on user profile. Needs some design work on the tabs and the overview page.

Waiting on API changes from @kemitche to access trophies without oauth tokens.

:eyeglasses: @danehansen, @btholt

### CODE REVIEW

File: assets/less/components/dropdown.less [hunk 1]
```diff
@@ -10,6 +10,11 @@
     visibility: visible;
   }
 }
+  .Dropdown.Dropdown-inline {
```
Comment (btholt) [inline assets/less/components/dropdown.less]: I'm assuming the semantic meaning of the capital letter is that it coincides with a React component? If so, that's a rad idea.

Comment (ajacksified) [inline assets/less/components/dropdown.less]: Yup, that's a @danehansen thing.

File: src/views/pages/userGild.jsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+          <div className='well well-lg'>
+            <p>Sorry, this isn't ready yet!</p>
+          </div>
+        </div>
+      </main>
+    );
+  }
+
+  static populateData (api, props, synchronous) {
+    var defer = q.defer();
```
Comment (btholt) [inline src/views/pages/userGild.jsx]: I thought we were going with native promises? Q is fine; I just think we should be consistent.

Comment (ajacksified) [inline src/views/pages/userGild.jsx]: We've discussed it, but we haven't done the refactoring yet to get there in `reddit-mobile` and `snoode`. That'll follow at some point.

**Comment by btholt** [comment]
:fish: I'm fine with it as is. I think mixing types of promise may eventually come back to haunt this project, but maybe not.

File: src/views/pages/userGild.jsx (hunk 2 above)
Comment (ajacksified) [inline src/views/pages/userGild.jsx]: Edit: found some native promises, I'll go ahead and swap this out, it's at least in the right direction then.

Comment (ajacksified) [inline src/views/pages/userGild.jsx]: Doubleedit: since all of the pages use this pattern, I'll refactor them all at once in another pr.

**Pull request merged**